selftests/functional/plugin/test_assets.py

```diff
@@ -72,7 +72,7 @@ def test_asset_fetch_success(self):
             locations='https://mirrors.kernel.org/gnu/hello/hello-2.9.tar.gz')
         """
         test_content = TEST_TEMPLATE.format(content=fetch_content)
-        test_file = tempfile.NamedTemporaryFile(suffix=".py", delete=False)
+        test_file = tempfile.NamedTemporaryFile(suffix=".py", prefix=".avocado-selftest", delete=False)
```

Comment:
For most tests based on the `TestCaseTmpDir`, it should better to put temporary files under its `tmpdir`, and with that, it doesn't need a prefix or even be a temporary file.  The exception is if you need two distinct set of directories that can not be shared.